docs/guides/game-servers/how-to-install-amp-for-game-server-management/index.md

```diff
@@ -0,0 +1,114 @@
+---
+slug: installing-amp-game-server-management-panel
+author:
+  name: Ryan Syracuse
+  email: docs@linode.com
+description: "AMP or Application Management Panel is a game server management panel. It provides a graphical overlay to simplify the deployment of game servers, and is additionally the next iteration of the Minecraft server control panel McMyAdmin. AMP provides support for third party mods, heavy focus on security and a sleek web interface for managing your server. This guide covers the installation and configuration of a new AMP server on most Linux distributions."
```

Comment:
We need to get this description down to 155-160 characters. Maybe "AMP is a game server management panel. The successor to McMyAdmin, it provides a web interface to install games, mods, and increases server security."